contentScript.js

```diff
@@ -15,24 +42,26 @@ const cpm = wpm * 5;
 const timeToWrite = 60 / cpm;
 const text = lorem.replace(/\\n/, '');
 
-const writeChar = (char) => {
-    document.dispatchEvent(new KeyboardEvent("keydown", { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));
-    document.dispatchEvent(new KeyboardEvent("keypress", { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));
+const writeChar = (element, char) => {
+    element.dispatchEvent(new KeyboardEvent('keydown', { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));
+    element.dispatchEvent(new KeyboardEvent('keypress', { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));
     
     document.execCommand('insertText', true, char);
     
-    document.dispatchEvent(new KeyboardEvent("keyup", { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));    
+    element.dispatchEvent(new KeyboardEvent('keyup', { key: char.charCodeAt(0), char, shiftKey: true, bubbles: true }));    
 }
 
-let index = 0;
-const interval = setInterval(() => {
-    console.log(text[index])
-    writeChar(text[index]);
-    ++index;
-    console.log(window.getSelection());
+const writeOnElement = (element) => {
+    let index = 0;
+    interval = setInterval(() => {
+        console.log(text[index])
+        writeChar(element, text[index]);
+        ++index;
+        console.log(window.getSelection());
```

Comment:
can you please remove these console logs? (L57), I don't think they're useful anymore if we want to debug something we can add it locally.